info: Connected to the database. {"timestamp":"2024-05-10T06:11:57.961Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T06:13:40.907Z"}
info: Connected to the database. {"timestamp":"2024-05-10T06:13:41.308Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T06:14:30.492Z"}
info: Connected to the database. {"timestamp":"2024-05-10T06:14:30.849Z"}
error: 
    status - 500 
    message - Error: 
    at Query.run (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\sequelize\src\dialects\mssql\query.js:128:25)
    at C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\sequelize\src\sequelize.js:650:28
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at MSSqlQueryInterface.select (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\sequelize\src\dialects\abstract\query-interface.js:1001:12)
    at Function.findAll (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\sequelize\src\model.js:1824:21) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2024-05-10T06:17:39.348Z"}
error: 
    status - 500 
    message - Error: 
    at Query.run (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\sequelize\src\dialects\mssql\query.js:128:25)
    at C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\sequelize\src\sequelize.js:650:28
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at MSSqlQueryInterface.select (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\sequelize\src\dialects\abstract\query-interface.js:1001:12)
    at Function.findAll (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\sequelize\src\model.js:1824:21) 
    url - /api/v1/users/ 
    method - POST 
    IP - ::1
   {"timestamp":"2024-05-10T06:18:13.783Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T06:20:12.859Z"}
info: Connected to the database. {"timestamp":"2024-05-10T06:20:13.256Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T06:24:03.526Z"}
info: Connected to the database. {"timestamp":"2024-05-10T06:24:03.867Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T06:26:16.605Z"}
info: Connected to the database. {"timestamp":"2024-05-10T06:26:16.998Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T06:29:15.091Z"}
info: Connected to the database. {"timestamp":"2024-05-10T06:29:15.284Z"}
error: 
    status - 500 
    message - ReferenceError: data is not defined
    at _callee6$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\services\/user.service.js:63:33)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2024-05-10T06:29:18.334Z"}
error: 
    status - 500 
    message - ReferenceError: data is not defined
    at _callee6$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\services\/user.service.js:63:33)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2024-05-10T06:29:39.069Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T06:29:53.174Z"}
info: Connected to the database. {"timestamp":"2024-05-10T06:29:53.357Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:24:26.552Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:25:11.014Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:25:17.327Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:28:33.089Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:32:30.382Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:33:13.629Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:33:31.631Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:35:13.131Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:35:19.255Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:35:32.522Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:39:25.389Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:40:20.123Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:40:26.341Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:41:02.280Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:41:19.473Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:41:25.986Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:41:51.189Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:41:57.327Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:42:12.939Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:42:19.886Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:42:26.283Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:42:38.359Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:44:45.929Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:44:56.550Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:45:17.584Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:46:18.548Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:46:30.552Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:47:02.333Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:47:37.949Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T07:47:52.683Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:47:53.083Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:49:02.779Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T07:52:01.549Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:52:01.929Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:53:44.571Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:55:18.281Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T07:55:40.415Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:55:40.747Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T07:56:11.753Z"}
info: Connected to the database. {"timestamp":"2024-05-10T07:56:12.097Z"}
error: 
    status - 500 
    message - Error: User is not present
    at _callee5$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\services\/user.service.js:54:11)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2024-05-10T07:57:53.558Z"}
error: 
    status - 500 
    message - Error: User is not present
    at _callee5$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\services\/user.service.js:54:11)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2024-05-10T07:58:33.362Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T08:00:25.968Z"}
info: Connected to the database. {"timestamp":"2024-05-10T08:00:26.371Z"}
error: 
    status - 500 
    message - Error: User is not present
    at _callee5$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\services\/user.service.js:54:11)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2024-05-10T08:00:44.043Z"}
error: 
    status - 500 
    message - Error: User is not present
    at _callee5$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\services\/user.service.js:54:11)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2024-05-10T08:01:45.180Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T08:02:10.493Z"}
info: Connected to the database. {"timestamp":"2024-05-10T08:02:10.859Z"}
error: 
    status - 500 
    message - Error: User is not present
    at _callee5$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\services\/user.service.js:54:11)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2024-05-10T08:02:23.771Z"}
error: 
    status - 500 
    message - Error: User is not present
    at _callee5$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\services\/user.service.js:54:11)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2024-05-10T08:05:37.089Z"}
error: 
    status - 500 
    message - Error: User is not present
    at _callee5$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\services\/user.service.js:54:11)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/employee 
    method - GET 
    IP - ::1
   {"timestamp":"2024-05-10T08:08:21.392Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T08:08:54.218Z"}
info: Connected to the database. {"timestamp":"2024-05-10T08:08:54.722Z"}
error: 
    status - 500 
    message - TypeError: employeeservice.createEmployee is not a function
    at _callee$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\controllers\/employee.controller.js:5:44)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at createEmployee (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\controllers\/employee.controller.js:3:28)
    at Layer.handle [as handle_request] (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\middlewares\/auth.middleware.js:27:5)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/employee/ 
    method - GET 
    IP - ::1
   {"timestamp":"2024-05-10T08:09:04.664Z"}
error: 
    status - 500 
    message - TypeError: employeeservice.createEmployee is not a function
    at _callee$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\controllers\/employee.controller.js:5:44)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at createEmployee (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\controllers\/employee.controller.js:3:28)
    at Layer.handle [as handle_request] (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\src\middlewares\/auth.middleware.js:27:5)
    at tryCatch (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\rajsh\Desktop\BridgeLabs\Node\NodeProject\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/employee/ 
    method - GET 
    IP - ::1
   {"timestamp":"2024-05-10T08:10:28.028Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T08:10:49.310Z"}
info: Connected to the database. {"timestamp":"2024-05-10T08:10:49.634Z"}
info: Connected to the database. {"timestamp":"2024-05-10T09:51:27.573Z"}
info: Connected to the database. {"timestamp":"2024-05-10T09:53:12.741Z"}
info: Connected to the database. {"timestamp":"2024-05-10T09:54:09.645Z"}
info: Connected to the database. {"timestamp":"2024-05-10T09:54:33.967Z"}
info: Connected to the database. {"timestamp":"2024-05-10T09:55:48.833Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T09:56:50.407Z"}
info: Connected to the database. {"timestamp":"2024-05-10T09:56:50.784Z"}
info: Connected to the database. {"timestamp":"2024-05-10T09:57:33.829Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T09:59:39.968Z"}
info: Connected to the database. {"timestamp":"2024-05-10T09:59:40.338Z"}
error: 
    status - 500 
    message - ValidationError: "notes" must be in iso format 
    url - /api/v1/employee/ 
    method - POST 
    IP - ::1
   {"timestamp":"2024-05-10T10:02:46.788Z"}
error: 
    status - 500 
    message - ValidationError: "notes" must be in iso format 
    url - /api/v1/employee/ 
    method - POST 
    IP - ::1
   {"timestamp":"2024-05-10T10:04:33.252Z"}
info: Server started at localhost:3000/api/v1/ {"timestamp":"2024-05-10T10:04:54.171Z"}
info: Connected to the database. {"timestamp":"2024-05-10T10:04:54.590Z"}
error: 
    status - 500 
    message - ValidationError: "start_date" must be in iso format 
    url - /api/v1/employee/ 
    method - POST 
    IP - ::1
   {"timestamp":"2024-05-10T10:04:58.459Z"}
error: 
    status - 500 
    message - ValidationError: "notes" length must be at least 8 characters long 
    url - /api/v1/employee/ 
    method - POST 
    IP - ::1
   {"timestamp":"2024-05-10T10:06:48.446Z"}
